deployments/cloudfoundry/bosh/jobs/splunk-otel-collector/templates/splunk-otel-collector.conf.erb

```diff
@@ -1,9 +1,10 @@
 <% if p('otel.proxy.http', '').length > 0 %>
 export HTTP_PROXY=<%= p('otel.proxy.http') %>
+export http_proxy=<%= p('otel.proxy.http') %>
```

Comment:
Some architectures expect this in lower case (`http_proxy`), so I've added it as a safe-guard here.